lib/components/component.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:ui';
 
+import 'package:flame/effects/effects.dart';
```

Comment:
We have been using relative imports for local files.

Reply:
Stupid IDE, I'll update it. :)